setup.py

```diff
@@ -28,7 +28,7 @@
     "scipy",
     "sklearn",
     "parameterized",
-    "deepspeed",
+    "deepspeed==0.6.5",
```

Comment:
```suggestion
    "deepspeed<=0.6.5",
```
I don't think we required 0.6.5 as a min version before, so let's not pin exactly.